### ISSUE TITLE
Add SQL based layers for schema and tables

### ISSUE THREAD

**Issue by Gustry** [3liz/qgis-pgmetadata-plugin#9]
In the administration project, we should add `schemas` and `tables` which would be based on SQL queries 👍🏻 

```sql
SELECT schema_name
FROM information_schema.schemata
WHERE schema_name NOT IN ('pg_toast', 'pg_temp_1', 'pg_toast_temp_1', 'pg_catalog', 'information_schema');
```

```sql
SELECT table_schema, table_name
FROM information_schema.tables
WHERE table_schema NOT IN ('pg_toast', 'pg_temp_1', 'pg_toast_temp_1', 'pg_catalog', 'information_schema');
```

And use these tables in the form for auto-completion.

**Comment by Gustry** [comment]
The idea is to have two layers loaded in QGIS : 
* listing schemas
* listing tables (with its schema)

* Configure the QGIS form in dataset to 
  * Show the dropdown list of schema, user can add a new value if needed
  * Filter the list of tables when the schema is set, user can add a new value if needed

@drillinP Can you have a look to this ticket ?

**Comment by Gustry** [comment]
Fixed with #21 and c6a688fc50ea3007f37b6f36f3651a933ed86c70
